optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -647,12 +662,20 @@ def best_trial(self) -> optuna.trial.FrozenTrial:
 
 
 class LightGBMTuner(LightGBMBaseTuner):
-    """Hyperparameter-tuning with Optuna for LightGBM.
+    """Hyperparameter tuner for LightGBM.
+
+    It optimizes the following hypeperparameters in stepwise manner:
+    ``lambda_l1``, ``lambda_l2``, ``num_leaves``, ``feature_fraction``, ``bagging_fraction``,
+    ``bagging_freq`` and ``min_child_samples``.
+
+    You can find the detailed algorithm and benchmark results in `this blog article <https://mediu
```

Comment:
"the details of the algorithm" is more correct I think.

Reply:
Thanks! fixed: https://github.com/optuna/optuna/pull/1217/commits/7dd413b89954a598c935d6d513b18864aa8fb890.